docs/public/components/testing.md

```diff
@@ -1,3 +1,210 @@
 # Testing components
```

Comment:
I think one of the main focus of this doc should be to explain how to create the `mock` implementation and how to link mocks together.

Reply:
I would argue that would be a separate page rather the one for testing